syntax = "proto3";
option cc_generic_services = true;
//not neccesary
//import "sofa/pbrpc/rpc_option.proto";
message FooRequest{
    int32 code = 1;
}
message FooResponse{
    string text = 1;
}
service FooService{
    rpc Foo(FooRequest) returns(FooResponse);
}